editor/src/components/canvas/commands/add-element-command.ts

```diff
@@ -63,12 +69,15 @@ export const runAddElement: CommandFunction<AddElement> = (
         underlyingFilePathNewParent,
       )
 
-      editorStatePatches = [editorStatePatchNewParentFile]
+      editorStatePatches = [
+        editorStatePatchNewParentFile,
+        includeToastPatch(insertionResult.insertionDetails, editorState),
+      ]
     },
   )
 
   return {
     editorStatePatches: editorStatePatches,
-    commandDescription: `Add Element to ${EP.toString(command.parentPath)}`,
+    commandDescription: `Add Element to ${JSON.stringify(command.parentPath)}`,
```

Comment:
maybe it'd be worth to add a utility function to convert `ReparentTargetParent` to string

Reply:
Fixed in e1cd2a6b4.